.changeset/thirty-papayas-attack.md

```diff
@@ -0,0 +1,36 @@
+---
+'@backstage/techdocs-common': minor
+'@backstage/plugin-techdocs-backend': minor
```

Comment:
These changesets seem switched around, assuming this one should be for `create-app` since it has be description for how to migrate?

Reply:
Hmm, interesting. I thought these migration instructions make sense in the `techdocs-backend` changelogs for people who are using this plugin, and want to upgrade.

For `create-app`, I imagine the changelog to be just a FYI about the newer features that come with using the latest create-app package to create new app.

Happy to copy the migration instruction to `create-new-app` if you say so.

Reply:
Idea is to use the `create-app` changelog as a method for keeping existing apps up to date with changes done to the template. You could just add `create-app` to this changeset and remove the other one tbh

Reply:
Noted. Done @Rugvip